src/js/_store/modules/categories.js

```diff
@@ -42,5 +69,32 @@ export default {
         commit('failRequest', { message: err.message });
       });
     },
+    editedCategory({ commit }, category) {
+      commit('editedCategory', category);
+    },
+    updateCategory({ commit, rootGetters, state }) {
```

Comment:
[imo]

カテゴリーの新規作成になるので、以下が適切かと思いました！

```suggestion
    postCategory({ commit, rootGetters, state }) {
```